docs/resources/azurerm_webapp.md.erb

```diff
@@ -0,0 +1,125 @@
+---
+title: About the azurerm_webapp Resource
+platform: azure
+---
+
+# azurerm\_webapp
+
+Use the `azurerm_webapp` InSpec audit resource to test properties of an Azure Webapp.
+
+<br />
+
+## Azure REST API version
+
+This resource interacts with version `2016-08-01` of the Azure Management API.
+For more information see the [official Azure documentation](https://docs.microsoft.com/en-us/rest/api/appservice/webapps/get).
+
+At the moment, there doesn't appear to be a way to select the version of the
+Azure API docs. If you notice a newer version being referenced in the official
+documentation please open an issue or submit a pull request using the updated
+version.
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+## Syntax
+
+An `azurerm_webapp` resource block identifies an AppService Webapp by name and Resource Group.
+
+    describe azurerm_appservice_webapp(resource_group: 'example', name: 'webapp name') do
+      ...
+    end
+
+<br />
+
+## Examples
```

Comment:
Would be great to see examples of the additional resource methods added here, for example
```
its('auth_settings.properties') { should have_attributes(enabled: false) }
its('configuration.properties') { should have attributes(netFrameworkVersion: 'v4.0') }
```